scripts/count-bundles.py

```diff
@@ -47,8 +48,10 @@ def get_project_bundle_count(project_name):
 
                 all_bundle_ids.add(f"{bundle['bundleUuid']}.{bundle['bundleVersion']}")
 
-        search_after = response_json['pagination']['search_after']
-        search_after_uid = response_json['pagination']['search_after_uid']
+        pagination_params = extract_internal_pagination_params(response_json)
```

Comment:
This script is a client to Azul and so should just be using the next prev links and does not need to access the internals. If you eliminate the helper calls here the helper functions in `pagination.py` (the existence of which is a bad smell, btw) are only used in tests and can therefore be added as methods of a TestCase subclass.